### PR TITLE
Attributes are now a list

### PR DESCRIPTION
**Pull request by benoit-grg**

### CODE REVIEW

File: src/main/scala/com/ubirch/webui/batch/Batch.scala [hunk 1]
```diff
@@ -51,7 +51,7 @@ sealed trait Batch[D] {
       inputStream: InputStream,
       skipHeader: Boolean,
       description: String,
-      tags: String
+      tags: List[String]
```
Comment (Carlos-Augusto) [inline src/main/scala/com/ubirch/webui/batch/Batch.scala:54]: I would like to understand why was it required to change the signature of the field?

**Review by Carlos-Augusto — COMMENTED** [review]
I am curious as to what led to this modification.

File: src/main/scala/com/ubirch/webui/batch/Batch.scala (hunk 1 above)
Comment (benoit-grg) [reply src/main/scala/com/ubirch/webui/batch/Batch.scala:54]: True, I can cast it directly at
TAGS.name -> batchRequest.tags.split(",")
and not change the rest

Comment (Carlos-Augusto) [reply src/main/scala/com/ubirch/webui/batch/Batch.scala:54]: I am not saying it is a problem, I am just trying to get the basic idea behind it

**Comment by Carlos-Augusto** [comment]
I am also wondering how this change affects the UI and if so, has it been updated yet?

File: src/main/scala/com/ubirch/webui/batch/Batch.scala (hunk 1 above)
Comment (benoit-grg) [reply src/main/scala/com/ubirch/webui/batch/Batch.scala:54]: The idea was to change it directly from the ApiDevices 
It won't impact the web-ui as it's not displayed there.

Comment (Carlos-Augusto) [reply src/main/scala/com/ubirch/webui/batch/Batch.scala:54]: Understood

**Pull request merged**